biorbd_optim/plot.py

```diff
@@ -144,10 +144,11 @@ def create_plots(self, has, var_type):
 
     def _organize_windows(self, nb_windows):
         height = tkinter.Tk().winfo_screenheight()
-        width = tkinter.Tk().winfo_screenwidth()
-        self.nb_vertical_windows, nb_horizontal_windows = PlotOcp._generate_windows_size(nb_windows)
+        width = tkinter.Tk().winfo_screenwidth() if True else tkinter.Tk().winfo_screenwidth() / 2
+
```

Comment:
Is there no better way to get the number of monitors? I could not find one quickly myself (without external dependecies)..